Manager/UserManager.php

```diff
@@ -192,4 +221,42 @@ public function deleteUser(UserInterface $user)
         $this->entityManager->remove($user);
         $this->entityManager->flush();
     }
+
+    /**
+     * @param User $user
+     * @return mixed
+     * @throws TransportExceptionInterface
+     */
+    public function sendInvitation(User $user)
+    {
+        if (null !== $user) {
+            $fromEmail = $this->parameterBag->get('webstack_user.registration.from_email');
+
+            if (null === $user->getConfirmationToken()) {
+                $user->setConfirmationToken($this->tokenGenerator->generateToken());
+            }
+
+            $user->setPasswordRequestedAt(new DateTime());
+
+            $this->entityManager->flush();
+
+            $email = (new TemplatedEmail())
+                ->from(new Address($fromEmail['address'], $fromEmail['sender_name']))
+                ->to($user->getEmail())
+                ->subject('Bevestig uw account')
+                ->htmlTemplate('@WebstackUser/email/send-invitation/index.html.twig')
+                ->context([
+                    'user' => $user,
+                    'confirmationUrl' => $this->router->generate('webstack_user_reset_password_reset', [
+                        'token' => $user->getConfirmationToken()
+                    ], UrlGeneratorInterface::ABSOLUTE_URL)
+                ]);
+
+            $this->mailer->send($email);
+        }
+
+        return true;
+    }
```

Comment:
```suggestion
    }
```

return true is zinloos, want mislukt het dan hebben we al ergens een exception te pakken